ui-tests/registerFailure.spec.js

```diff
@@ -0,0 +1,285 @@
+import { test, expect } from "@playwright/test";
+import dotenv from "dotenv";
+import mongoose from "mongoose";
+import userModel from "../models/userModel";
+
+dotenv.config();
+
+test.beforeEach(async ({ page }) => {
+  // Connect to DB for deleting test user later
```

Comment:
No test user is created so maybe remove this comment

Reply:
good catch, thank you Douglas!